examples/py/binance-universal-transfer.py

```diff
@@ -14,14 +14,14 @@ def main():
 
     # apiKey must have universal transfer permissions
     binance = ccxt.binance({
-        "apiKey": "...",
-        "secret": "...",
+        "apiKey": "YOUR_API_KEY",
+        "secret": "YOUR_SECRET",
     })
     binance.load_markets()
 
     pprint(binance.transfer('USDT', 0.1, 'spot', 'future'))
-    transfers = binance.fetchTransfers()
-    pprint('got ' + str(len(transfers)) + ' transfers')
+    transfers = binance.fetch_transfers()
+    pprint('there is ' + str(len(transfers)) + ' transfers')
```

Comment:
in the future I would put the example edits, and the exchange edits in different PRs, seeing as they don't depend on each other